src/components/PlayerCard.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const playerCard = ({ player, removeFunc }) => {
+	return (
+		<li className='border bg-gray-300 flex justify-between px-3 rounded'>
+			{player} {removeFunc && <button onClick={() => removeFunc(player)}>x</button>}
```

Comment:
create a new line, these have a `display: flex` the styling won't change but be more clear to read

Reply:
i didn't quite get it, can you give an example?

Reply:
Just get the button logic in the line below, don't put it in the same line too messy for the eyes